models/instrument.py

```diff
@@ -1600,7 +1600,7 @@ def overscan_and_data_sections( self, header ):
         Parameters
         ----------
         header: fits.Header
-          The header of the image in question; use Image.raw_header not Image.header.
+          The header of the image in question; use Image.header not Image.info.
```

Comment:
With the names .header and .info instead of .header and .raw_header, the comment after the semicolon probably isn't needed any more.  (No harm leaving it in.)

Reply:
yeah, I agree.